test/features/get_open_api.feature

```diff
@@ -0,0 +1,53 @@
+# @method=GET @endpoint=/{GovStackInstance}/{memberClass}/{memberCode}/{applicationCode}/getOpenAPI
+Feature: Retrieve openAPI description of the specified REST service
+
+  Retrieve openAPI description of the specified REST service
+
+  @smoke
+  Scenario: Retrieve the openAPI description of the specified REST service smoke type test
+
+    Given User wants to retrieve the openAPI description of the specified REST service
+    When User sends GET request with given "identifier" as GovStackInstance "identifier" as memberClass "alp4aNum3r1c" as memberCode "alp4aNum3r1c" as applicationCode
+    Then User receives a resonse
+    And The response should match json schema
+    And The response should have status 200
+    And The response should be returned in a timely manner
```

Comment:
I suggest to change the order of AND. First "...should be returned in a timely manner", then check the response status, then header content-type or similar and at the end json schema :)